tools/c7n_org/c7n_org/cli.py

```diff
@@ -479,6 +481,7 @@ def run_account(account, region, policies_config, output_path,
                     account['name'], region, p.name, len(resources),
                     time.time() - st)
             except ClientError as e:
+                success = False
                 if e.response['Error']['Code'] == 'AccessDenied':
```

Comment:
need to `return policy_counts, success` in this block too at line 488 otherwise the accessdenied is being swallowed silently

Reply:
nice catch, updated to handle that.